.changeset/afraid-poets-change.md

```diff
@@ -0,0 +1,7 @@
+---
+"@graphql-mesh/types": patch
+"@graphql-mesh/transform-naming-convention": patch
+"@graphql-mesh/website": patch
```

Comment:
This might fail on release. Could you remove this line?

Reply:
Fixed 👍🏼